byte INFO=1
byte WARN=2
byte ERROR=4

std_msgs/Header header
byte level
string message
string[] nodes
string topic